exercises/forth/forth_test.go

```diff
@@ -0,0 +1,53 @@
+package forth
+
+// API:
+// func Forth([]string) ([]int, error)
+//
+
+import "testing"
+
+const targetTestVersion = 1
+
+type testCase struct {
+	description string
+	input       []string
+	expected    []int // nil slice indicates error expected.
+}
+
+func sliceEqual(a, b []int) bool {
```

Comment:
I will use DeepEqual.  I had not used reflect before.  But in doing so, I discovered an error in my example.go, where it was returning a nil slice instead of an empty slice. Glad that was caught. I found out that DeepEqual returns false for two slices a and b, if a is nil but b is an empty slice; however len(a) and len(b) are both zero, and a for loop ranging over either a or b will execute zero times, which was what my sliceEqual was doing.  Learning more about Go all the time.